fs/copy_test.go

```diff
@@ -6,7 +6,7 @@ import (
 
 	_ "crypto/sha256"
 
-	"github.com/docker/containerd/fs/fstest"
+	"github.com/docker/containerd/testutil/fstest"
```

Comment:
`testutil` itself was newly not added in this PR. Do you mean I should revert `testutil/fstest` to `fs/test`?